sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,11 +1,26 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (2020-08-11)
+
+- Added the optional `serviceVersion` property to the `CertificateClient` optional parameters.
+    - It defaults to the latest supported API version, which currently is `7.1`.
+    - Other supported service version at the moment is `7.0`.
+    - If you were using `4.1.0-preview.1` version of this package, then this was previously named `apiVersion`.
+- `CertificateKeyType` now supports the `oct` type.
+    - Since the service now supports `oct` keys, now the client will correctly report the type of these keys.  
```

Comment:
Is there any service documentation we can link to for getting more details on what this `oct` key is? I saw somewhere this being referred to as `Octet`. What is the right term to use?